google/cloud/apigee/registry/applications/v1alpha1/registry_manifest.proto

```diff
@@ -46,7 +46,7 @@ message GeneratedResource {
 	];
 	string filter = 2;
 	//set to true if the action is not going to store any information in the GeneratedResource
-	bool placeholder = 3;
+	bool receipt = 3;
```

Comment:
Could we make this "is_receipt" or something similar? (cc @seaneganx for recommendations) 

I think "receipt" by itself suggests that the receipt is true or false (a boolean) and what we mean here is that the generated resource is a receipt.

Reply:
Using an "is_" prefix is recommended against in [AIP-140](https://google.aip.dev/140#booleans). I think adding documentation to the field will be sufficient because it's an obvious location to seek clarification if someone doesn't have enough context.

Note: Some people (myself included) are already going to read this automatically as "`GeneratedResource` is a `receipt`". If someone wanted to suggest that the receipt is a bool itself, I would expect them to have a `Receipt` message with a field `value`.

Reply:
OK - receipt SGTM

Reply:
I updated the comment to make it more clear  (the purpose of the receipt field)